src/interface/storage.thrift

```diff
@@ -428,6 +428,22 @@ struct DropCPRequest {
     2: string                       name,
 }
 
+struct RebuildTagIndexRequest {
+    1: common.GraphSpaceID          space_id,
+    2: list<common.PartitionID>     parts,
+    3: common.TagID                 tag_id,
+    4: common.SchemaVer             tag_version,
+    5: common.IndexID               index_id,
+}
```

Comment:
A small suggestion, the following elements are enough，WDYT？
struct RebuildTagIndexRequest {
    1: common.GraphSpaceID          space_id,
    2: list<common.PartitionID>     parts,
    3: common.IndexID               index_id,
}

Reply:
`tag_id` and `tag_version` will use to skip some data